dnf/module/module_base.py

```diff
@@ -73,7 +73,7 @@ def install(self, module_specs, strict=True):
         # <package_name, set_of_spec>
         fail_safe_repo = hawkey.MODULE_FAIL_SAFE_REPO_NAME
         install_dict = {}
-        install_set_artefacts = set()
+        install_set_artifacts = set()
```

Comment:
I'd prefer you squashed this commit with the previous one, fixing an improperly selected name is just a noise in git history (it's a good practice to maintain a clean set of commits in a PR)

Reply:
I agree, but some may be most of the changes are in code that was not created in this PR. But I can modify it, but commit will remain.

Reply:
I see, I didn't notice it was there before. It would still be a bit cleaner if you fixed up the other commit, but I don't insist.

Reply:
I fixed already other commits.